README.md

```diff
@@ -10,135 +10,183 @@ Follow the following instructions to get the project ready to use ASAP.
 
 ### Requirements
 
-Be sure you have installed in your system
+Be sure you have installed in your system:
 
 - [Python version 3](https://www.python.org/download/releases/3.0/) (recommended 3.8 or less) in your path. It will install
   automatically [pip](https://pip.pypa.io/en/stable/) as well.
-- A virtual environment, namely [venv](https://docs.python.org/3/library/venv.html).
+- A virtual environment, namely [.venv](https://docs.python.org/3/library/venv.html).
 - Optionally for running Azure functions locally: [Azure functions core tool](https://docs.microsoft.com/en-us/azure/azure-functions/functions-run-local?tabs=macos%2Ccsharp%2Cbash).
 
+- Docker
+  
+  You can follow the instructions below to install on each of the following operating systems:
+  - [**Mac**](https://docs.docker.com/docker-for-mac/install/)
+  - [**Linux**](https://docs.docker.com/engine/install/)
+  - [**Windows**](https://docs.docker.com/docker-for-windows/install/)
+
+- Docker Compose
+
+  To install Docker Compose, please choose the operating system you use and follow the steps [here](https://docs.docker.com/compose/install/).
+
 ### Setup
 
-- Create and activate the environment,
+Once installed Docker and Docker Compose we must create a `.env` file in the root of our project where we will put the following environment variables.
 
-  In Windows:
+```shell
+export MS_AUTHORITY=XXXX
+export MS_CLIENT_ID=XXXX
+export MS_SCOPE=XXXX
+export MS_SECRET=yFo=XXXX
+export MS_ENDPOINT=XXXX
+export DATABASE_ACCOUNT_URI=XXXX
+export DATABASE_MASTER_KEY=XXXX
+export DATABASE_NAME=XXXX
+export FLASK_APP=XXXX
+export AZURE_APP_CONFIGURATION_CONNECTION_STRING=XXXX
+export FLASK_DEBUG=XXXX
+export REQUESTS_CA_BUNDLE=XXXX
+```
+> **Please, contact the project development team for the content of the variables mentioned above.**
 
-  ```
-  #Create virtual enviroment
-  python -m venv .venv
+### Run containers
 
-  #Execute virtual enviroment
-  .venv\Scripts\activate.bat
-  ```
+Once all the project configuration is done, we are going to execute the following command in the terminal, taking into account that we are inside the root folder of the project:
 
-  In Unix based operative systems:
+```shell
+docker-compose up --build
+```
 
-  ```
-  #Create virtual enviroment
-  virtualenv .venv
+This command will build all images with the necessary configurations for each one, aslo
```

Comment:
At the end of the line you wrote "aslo", I think the correct word is "also".

Reply:
My mistake, I will correct it, thank you very much for checking.